src/Microsoft.Framework.Runtime.Hosting/Dependencies/ReferenceAssemblyDependencyProvider.cs

```diff
@@ -31,7 +32,13 @@ public ReferenceAssemblyDependencyProvider(IFrameworkReferenceResolver framework
 
         public Library GetDescription(LibraryRange libraryRange, NuGetFramework targetFramework)
         {
-            System.Diagnostics.Debug.Assert(SupportsType(libraryRange.TypeConstraint));
+            Debug.Assert(SupportsType(libraryRange.TypeConstraint));
+
+            if(!targetFramework.IsDesktop())
```

Comment:
Ok, I'll also update the FrameworkReferenceResolver to only rewrite DNX frameworks to Net.